### PR TITLE
Timer &  ui fix

### PR DESCRIPTION
**Pull request by TSharma30**

### CODE REVIEW

File: client/src/App.jsx
Comment (4kshi7) [inline client/src/App.jsx:1]: name change

File: client/src/components/Cursor/Cursor.jsx
Comment (4kshi7) [inline client/src/components/Cursor/Cursor.jsx:1]: cursor fix on medium and small screens

File: client/src/components/Home/Home.jsx
Comment (4kshi7) [inline client/src/components/Home/Home.jsx:1]: padding fix

**Comment by TSharma30** [comment]
cursor fixed for medium and small screens

File: client/src/components/Register/Register.jsx
Comment (4kshi7) [inline client/src/components/Register/Register.jsx:1]: label fixes

File: client/src/index.css
Comment (4kshi7) [inline client/src/index.css:1]: bg fixes for mobile screen

**Review by 4kshi7 — APPROVED** [review]
Approving this pr for ui changes

**Pull request merged**